### ISSUE TITLE
vagrant up failed

### ISSUE THREAD

**Issue by Exoskill** [kleiram/vagrant-symfony#31]
err: /Stage[main]/Phpqatools/Pear::Package[File_Iterator]/Package[pear-pear.phpunit.de-File_Iterator]/ensure: change from absent to latest failed: Could not update: Execution of '/usr/bin/pear channel-discover pear.phpunit.de' returned 1: Discovering channel pear.phpunit.de over http:// failed with message: channel-add: Cannot open "http://pear.phpunit.de/channel.xml" (File http://pear.phpunit.de:80/channel.xml not valid (received: HTTP/1.1 410 Gone

**Comment by mablae** [comment]
I think the PHP unit pear channel is closed now by start of 2016: 

PhpUnit should be installed using composer or phar package archive instead.

**Comment by kleiram** [comment]
Yeah, that's probably it... should be a quick fix! Maybe I have some time left this week..
